packages/salesforcedx-vscode-core/src/DeveloperLogTraceFlag.ts

```diff
@@ -62,14 +62,8 @@ export class DeveloperLogTraceFlag {
     this.prevVFDebugLevel = oldVFDebugLevel;
```

Comment:
Why is this file at the top-level of src?

Reply:
@Sweetman - Ping. Just checking. Do you want this file here at the top-level?

Reply:
I see a new file so maybe you just need to delete this file properly to make sure it's no longer there.